vars/jobs.groovy

```diff
@@ -38,3 +38,16 @@ def build_job(name, parameters, is_stable)
   }
   return job
 }
+
+def executor(name, parameters, is_runner=false){
```

Comment:
is_runner parameter could be more general like ignore_unstable or dont_notify_unstable

Reply:
Also maybe I would name the function like build_and_notify ? WDYT? To let us know the purpose of this function from its name.